delira/models/classification/classification_network_tf.py

```diff
@@ -181,11 +188,7 @@ def closure(model: typing.Type[AbstractTfNetwork], data_dict: dict,
 
             image_names = "val_" + str(image_names)
 
-        for key, val in {**metric_vals, **loss_vals}.items():
-            logging.info({"value": {"value": val.item(), "name": key,
-                                    "env_appendix": "_%02d" % fold
-                                    }})
 
         #logging.info({'image_grid': {"image_array": inputs, "name": image_names,
         #                             "title": "input_images", "env_appendix": "_%02d" % fold}})
-        return metric_vals, loss_vals, [preds]
+        return metric_vals, loss_vals, {"pred": preds}
```

Comment:
is this still required? As far as i can see, the dict structure is only required to pass it through Predictor._convert_batch_to_npy(). My outputs are now naturally of dict type and therefore I would pass the whole returned dict through here.

Reply:
No, this isn't required, This was only my first try to make it work with TF :D